tests/com/twitter/intellij/pants/settings/PantsProjectSettingsTest.java

```diff
@@ -7,9 +7,10 @@
 import com.intellij.ui.CheckBoxList;
 import com.intellij.util.containers.ContainerUtil;
 import com.twitter.intellij.pants.testFramework.OSSPantsImportIntegrationTest;
-import org.fest.util.Sets;
```

Comment:
same goes for guava sets for ease of use.

Reply:
:)